pkg/segment/query/querystatus.go

```diff
@@ -1164,3 +1179,15 @@ func InitScrollFrom(qid uint64, scrollFrom uint64) error {
 
 	return nil
 }
+
+func ConvertQueryCountToTotalResponse(qc *structs.QueryCount) interface{} {
```

Comment:
Looks like when you moved this function, you also changed it to always return a `HitsCount` type. I think that's a good change, but you should also change the return type from `interface{}` now

Reply:
addressed in the latest commit.